### PR TITLE
Rs 2022.09.20

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-09-20-233354_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 1]
```diff
@@ -83,21 +83,23 @@ Actually, no, I'm mad.
 [sha:56cbf513476f551aff76d02eb5f301c6166acb0f]{
 -- Page 438, Offset 15850.
 -- 「……やっぱりな。分かってないと思った。
-"And just as I thought, you have no idea why.
+"Welp, guess I called it.
+ I figured you wouldn't understand."
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:87]: This sentence isn't nearly as snappy now, even if its a modicum more accurate.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 2]
```diff
@@ -181,62 +185,62 @@ I don't think I did anything that justifies getting so worked up."
...
 -- Page 447, Offset 15871.
 -- ちょっ、待って、どこ行くの志貴!?　話、まだ終わってないよ!?」
-Hey, wait, where are you going, Shiki?! I'm not done talking yet?!"
+Hey, wait, where are you going, Shiki?! I'm not done talking yet!"
 }
 [sha:59b0ede2509e23126f0d523291275b29e980fbcb]{
 -- Page 448, Offset 15872.
 -- おお。足が無意識に帰路についていた。
-Whoa, my legs unconsciously started making themselves their way home.
+Whoa, my legs just started making their own way home.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:221]: This sentence feels off somehow.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 3]
```diff
@@ -181,62 +185,62 @@ I don't think I did anything that justifies getting so worked up."
...
- it's not like I could get away from her,
- so I plotter back over to Arcueid's side.
+Though... it's not like I'd actually be able to give her the slip,
+ so I trudge back over to where Arcueid is standing.
 }
 [sha:b4f838845f5865d09dded09f17aa2c73798ce960]{
 -- Page 449, Offset 15875.
 -- 「わるい、考え事をしていたら足が勝手に動いた。
-"My bad, as I was thinking my legs took a life on their own.
+"My bad, as I was thinking my legs took on a life on their own.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:237]: Wouldn't it be "took on a life **of** their own?"

Though I suppose it's my bad for the messy writing to begin with.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 4]
```diff
@@ -181,62 +185,62 @@ I don't think I did anything that justifies getting so worked up."
...
 [sha:ef0710865eb2ea80053f6c308934c7a761cef8d8]{
 -- Page 449, Offset 15876.
 -- あと、玄関からやってきたらその時こそ絶交だからな。本気で恨んでやるから覚悟しやがれ」
-// the swearing is for やがれ
-Also,
- if the times comes that you visit by the front gate,
- I will actually cut off my relations with you.
- You better be goddamned prepared to bear the brunt of my resentment."
+Also, if you ever come traipsing in through the front gate, we're through.
+ I'm not screwing around here, if you try it you'd better be ready for me to curse your name."
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:243]: > "I'm not screwing around here, if you try it you'd better be ready for me to curse your name."

Feels like its lacking the impact of the original sentence.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 5]
```diff
@@ -316,22 +320,20 @@ Perhaps because my reply left little room for debate,
...
+"Well... I guess I went a bit too far as well.
 }
 [sha:d6a0e0ef3fbc75fc3d5999ce32d991a1bbfe7c21]{
 -- Page 455, Offset 15890.
 -- とにかく、今後はあんな無茶は止めてくれ。俺の心臓が止まりかねない」
-In any case,
- please give it a rest with all the crazy stuff from now on,
- or I might have a heart attack."
+Point is, please give it a rest with all the crazy stuff.
+ You're gonna give me a heart attack otherwise."
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:329]: Feel like the original is better

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 6]
```diff
@@ -382,13 +388,14 @@ But what's with the change of heart? Wasn't Vlov your only enemy?"
 -- Page 461, Offset 15901.
 -- 「？　わたしの敵はヴローヴじゃないよ？
 // ？ is voiced
-"Huh? My enemy wasn't Vlov though.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:385]: 'though' was to get the nuance of 'ないよ？' through. I still feel like it works better here.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 7]
```diff
@@ -382,13 +388,14 @@ But what's with the change of heart? Wasn't Vlov your only enemy?"
...
+"Huh? Vlov wasn't my enemy.
 }
 [sha:ad016f3eff1d767ca1716d36a98e40300e207c51]{
 -- Page 461, Offset 15902.
 -- わたしが探していたのはこの街に巣くった吸血鬼で、ヴローヴは関係ないもの」
 // from here on, much of the conversation mirrors stuff from ARC07_3D
-The vampire I was looking for lurks in this city――Vlov was unrelated"
+I was looking for a different vampire that lurked in this city,
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:397]: Revert; Vlov didn't actually lurk in the city.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 8]
```diff
@@ -490,75 +498,78 @@ Who on earth would go out of their way to do that?"
...
 -- Page 469, Offset 15922.
 -- アルクェイドの声には隠しようのない嫌悪が込められていた。
-There's a disgust in Arcueid's voice that she cannot conceal.
+There's an unmistakable note of disgust in Arcueid's voice.
 }
 [sha:ea7d59074211a417e62c22f2218d6b29b4606804]{
 -- Page 469, Offset 15923.
 -- ……一瞬、背筋が寒くなったのは、そんな彼女の声に気圧されたからだろう。
-For an instant, chills travel up my spine――her voice being like that is probably too much for me to take.
+Her voice is so overpowering that I feel a brief shiver run down my spine.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:520]: Perhaps oversimplified.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 9]
```diff
@@ -739,50 +750,51 @@ A grim foreboding crawls up my spine, and I my heart gives off a loud pulsation.
...
 -- Page 487, Offset 15965.
 -- アルクェイドは甘えた声で俺の名前を口にした。
-Arcueid calls my name with an almost sweet tenor.
+Arcueid calls my name in a wheedling tone.
 }
 [sha:d31b059ab9c682251682c75ea51fef62e4335238]{
 -- Page 487, Offset 15966.
 -- ……読めてきた。
-I saw this coming.
+...I can see where this is going.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:770]: Nah, the ellipsis works better at the end here, cmon.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 10]
```diff
@@ -739,50 +750,51 @@ A grim foreboding crawls up my spine, and I my heart gives off a loud pulsation.
...
 -- Page 488, Offset 15969.
 -- 「ロアを倒すのに力を使っちゃったから疲れちゃって。
-"I'm tired from having to use all that power to defeat Roa.
+"I ended up using a bunch of my strength to defeat Roa, so I'm a little worn out.
 }
 [sha:509adc7cd20b12744f0ff7ca7788b7916d95d9f2]{
 -- Page 488, Offset 15970.
 -- 誰かがわたしと一緒に、死徒の後始末をしてくれたら助かるんだけどなあ」
-Oh, if only someone could take care of these Dead Apostles with together with me, I would sooo appreciate it."
+Oh, if only there were %{i}someone%{/i} out there who'd come to my aid and help me take care of these Dead Apostles..."
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:791]: I think the previous version is both more accurate and reads better.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 11]
```diff
@@ -797,12 +809,12 @@ I do appreciate what Arcueid is doing.
...
 -- Page 489, Offset 15974.
 -- 街に残っている吸血鬼だって野放しにできない。
-The vampires who still remain in the city can't just be left alone.
+The vampires still in this city can't just be left to their own devices.
 }
 [sha:e740fc3a8a34463658193fe04d1bf750227e1068]{
 -- Page 490, Offset 15975.
 -- けど、だからといってコイツに手を貸すのは別問題だ。
-But still, that is a completely separate matter from me teaming up with her.
+Still, that doesn't mean I can just agree to help her on the spot.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:817]: 'on the spot' moves the problem from what it actually is. That makes it sound like his problem is that the current time and place is the problem, not the actual act of teaming up with her.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 12]
```diff
@@ -819,56 +831,56 @@ I've returned to my normal life.
...
+Arcueid might've flipped out when she spotted Ciel-senpai,
+ but at the end of the day she still helped us.
 }
 [sha:5055e925368ac447d9b143f4ad33971b870db683]{
 -- Page 494, Offset 15986.
 -- あの時アルクェイドが寒波を防いでくれなかったら、俺だけじゃなく街全体が氷漬けになっていた。
-If back then, Arcueid hadn't redirected the cold wave,
- then not only me, but the whole city would've frozen over.
+If she hadn't been there to protect us against the cold wave,
+ I'm not the only one who'd have ended up as a popsicle―――the whole city would have been frozen solid.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:878]: Is the popsicle comparison really necessary?

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 13]
```diff
@@ -880,7 +892,7 @@ Which means――now it's time for me to hold up my end of the bargain.
 -- Page 495, Offset 15989.
 -- いいよ、俺も吸血鬼の排除には賛成だし。怖いのはイヤだけど、我慢して協力してやる」
 Why not. I fully endorse getting rid of the vampires anyway.
- It's scary, and I don't like that, but I'll bear with it and cooperate."
+ I'm absolutely terrified, and I can't say I enjoy that one bit, but I'll stick it out and work together with you."
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:895]: This is just way too long.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 14]
```diff
@@ -895,23 +907,23 @@ I'm counting on you, Shiki! Let's fight together!"
...
+Arcueid is positively beaming.
 }
 [sha:fca4b6f43ea4ccff70a9b59678c75dba62979fe1]{
 -- Page 497, Offset 15993.
 -- ……頼りにするって、俺なんかいなくてもコイツひとりで充分だと思うけど……まあ、そう言ってくれるならやり甲斐はある。
-And who are you counting on, anyway...?
- I'm pretty sure she'd be totally fine without me.
- Though, her putting it like that does make it feel more worth doing.
+She's counting on me...?
+ I get the feeling that if I weren't here she'd be totally fine on her own.
```
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:916]: > She's counting on me...?
>  I get the feeling that if I weren't here she'd be totally fine on her own.

Both of these feel way better in the original. The second in particular is just unnecessarily long-winded and redundant for no particular reason.

**Review by Petrikow — COMMENTED** [review]
otu

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:221]: I dunno, reads pretty normally to me.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt [hunk 15]
```diff
@@ -341,37 +343,41 @@ Arcueid nods, a demure expression adorning her features.
 [sha:8a988d0b518c47f7af726270659f0566963800cf]{
 -- Page 457, Offset 15893.
 -- なんだ。ちゃんと注意すれば聞いてくれるじゃないか。
-Well, would you look at that. She does listen if you give her a proper reprimand.
+Well, would you look at that.
+ She actually will listen if you give her a stern warning.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_5.txt:347]: will in italics

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 7 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:397]: I mean, he might not have lurked for as long as Roa, but he _did_ have his lair which I'd say is worthy of the term "lurking"

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 10 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:791]: not sure I agree (ignoring the typo). I prefer the new version personally.

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 12 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:878]: I'm with Petrikow, feels like a weird liberty

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 13 above)
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:895]: prefer prev. version

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 7/07_01_CIEL07_5.txt (hunk 7 above)
Comment (Petrikow) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:397]: No, Vlov didn't actually have a lair.

This is misdirection from Nasu.

Comment (rschlaikjer) [reply script/Ciel/Day 7/07_01_CIEL07_5.txt:397]: I feel like lurking is pretty general 'up to no good' vibes, you don't necessarily have to set up shop.

**Pull request merged**